modules/porous_flow/test/tests/poroperm/PermFromPoroError.i

```diff
@@ -0,0 +1,199 @@
+# Testing permeability from porosity Error when A<0
```

Comment:
it'd be better to pass a command line argument to trigger the error than check in a whole input file that is not working by design and will show up in the documentation as an example

Reply:
I removed it and I added the errors to the inactive block in the material.